tests/test_buku.py

```diff
@@ -862,6 +884,28 @@ def test_copy_to_clipboard(platform, params):
             "markdown",
             "- [Untitled](http://example.com)\n- [Untitled](http://example.org)\n- [Google](http://google.com)\n",
         ],
+        [
+            "rss",
+            '<feed xmlns="http://www.w3.org/2005/Atom">\n'
+            "<title>Bookmarks</title>\n"
+            '<generator uri="https://github.com/jarun/buku">buku</generator>\n'
+            "<entry>\n"
+            "<title></title>\n"
+            '<link href="http://example.com" rel="alternate" type="text/html"/>\n'
+            "<id>1</id>\n"
+            "</entry>\n"
+            "<entry>\n"
+            "<title></title>\n"
+            '<link href="http://example.org" rel="alternate" type="text/html"/>\n'
+            "<id>1</id>\n"
+            "</entry>\n"
+            "<entry>\n"
+            "<title>Google</title>\n"
+            '<link href="http://google.com" rel="alternate" type="text/html"/>\n'
```

Comment:
…You may want to pick one kind of quotation marks and stick to it (instead of switching between them randomly).

Reply:
Just use `'` everywhere.

…Also, the doc structure will be more readable if you add spacing for visual indentation.

Reply:
Done